src/app/shared/entry/doi/manage-dois/manage-dois-dialog.component.html

```diff
@@ -29,9 +29,9 @@ <h1 mat-dialog-title>Manage DOIs</h1>
         Dockstore automatically creates a DOI for each valid, tagged version of this {{ entryTypeMetadata.term }} if it is published. It
```

Comment:
Whilst testing, I got tripped up by the "at least one author" requirement, which I wasn't aware of (not sure if I'd ever known or just forgotten).  So, maybe we can ad some text that alludes to it here, something like "has information about at least one author".

Reply:
Added that at least one author is required. Screenshots are updated in the PR description